src/test/cpp/subsystems/arm_test.cpp

```diff
@@ -0,0 +1,84 @@
+#include <frc/Filesystem.h>
+#include <gtest/gtest.h>
+#include <iostream>
+
+#include <filesystem>
+
+#include "frc/geometry/Pose2d.h"
+
+#include "subsystems/Arm.h"
+#include "io/ArmIO.h"
+#include "Parameters.h"
+
+class ArmTest : public testing::Test {
+public:
+    virtual void SetUp() {
+        arm_ = std::make_shared<Arm>(&arm_interface_);
+        arm_->Init();
+        // std::string deploy_path = frc::filesystem::GetDeployDirectory();
+        // std::string param_file = deploy_path + "/parameters.toml";
+        // RobotParams::LoadParameters(param_file);
```

Comment:
I'll work on getting the parameters working in CI, I think I learned something new recently